agent/stats/reporter/reporter.go

```diff
@@ -99,18 +109,23 @@ func NewDockerTelemetrySession(
 func (session *DockerTelemetrySession) Start(ctx context.Context) error {
 	backoff := retry.NewExponentialBackoff(time.Second, 1*time.Minute, 0.2, 2)
```

Comment:
nit & non-blocking comment. We can define some constants to configure the exp back-off behavior to make them less magical : )

Reply:
Will do.